tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4412,4 +4412,386 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+				]
+			)
+			->getMock();
+
+		$this->mock_gateway->method( 'get_return_url' )->willReturn( self::MOCK_RETURN_URL );
+
+		$this->mock_gateway->intent_controller = $this->getMockBuilder( WC_Stripe_Intent_Controller::class )
+			->onlyMethods( [ 'create_and_confirm_payment_intent', 'update_and_confirm_payment_intent', 'create_and_confirm_setup_intent' ] )
+			->getMock();
+
+		// Mock account data for fallback scenarios.
+		$this->set_stripe_account_data( $account_data );
```

PHPMD (2.15.0)

[error] The variable $account_data is not named in camelCase. (undefined)

(CamelCaseVariableName)